### DISCUSSION TITLE
Grafana Config/Backend with RAFT storage

### DISCUSSION THREAD

**Discussion by nicholas-cecere** [grafana/grafana#54791] [Platform]
The ability to use Raft Storage for Grafana's configuration back end.  This could be an option over using SQL of some kinda.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
